SuperblocksHQ/superblocks-lab
# Fix compiler's replyMessage

### Description of the Change
Fix compiler's replyMessage



### Benefits
Console logs replyMessage only once.



### Verification Process

1. Create any template
2. Open console and try to compile project
3. Check console, each contract should be logged only once as `[compiler] import file /contracts/....`



### Github Issues
Resolves #279
